samples/LargeResponseApp/LargeResponseApp.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web" ToolsVersion="15.0">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
It was required at the time we converted some of our repos, but with RC4 MSBuild not longer requires it.